controls-pyqt/gui.py

```diff
@@ -165,11 +161,11 @@ def __initialize_layout(self):
         
     def setup_ui(self):
         self.__create_window()
-        self.__initialize_labels()
+        self.__initialize_as_labels()
         
         self.__initialize_general_info()
         self.__initialize_imu_list()
-        self.__initialize_pwm_list()
+        self.__initialize_pwn_list()
```

Comment:
Wouldn't this be more accurate if it were `__initialize_pwm_list`?

Reply:
Took me too long to find the N. I copy and pasted the text from the function to solve this.